### PR TITLE
Fix password paths

### PR DESCRIPTION
**Pull request by KevinLiddle**
- pin -> code
- template indexes included

### CODE REVIEW

File: app/src/main/java/co/blustor/gatekeepersdk/services/GKAuthentication.java [hunk 1]
```diff
@@ -73,6 +69,32 @@ public AuthResult enrollWithFace(GKFaces.Template template, int templateId) thro
...
 
     /**
+     * Store a PIN at the first template index on the GateKeeper Card.
+     *
+     * @param pin the String to be stored as the password
+     * @return the {@code AuthResult} of the action
+     * @throws IOException when communication with the GateKeeper Card has been disrupted.
+     * @since 0.10.0
+     */
+    public AuthResult enrollWithPin(String pin) throws IOException {
```
Comment (esanmiguelc) [inline app/src/main/java/co/blustor/gatekeepersdk/services/GKAuthentication.java]: I think we are dropping the "PIN"

Comment (esanmiguelc) [inline app/src/main/java/co/blustor/gatekeepersdk/services/GKAuthentication.java]: "Recovery Code" is the one to use

**Comment by KevinLiddle** [comment]
@esanmiguelc updated all usages of "PIN" to "Recovery Code"

**Comment by esanmiguelc** [comment]
:+1:

**Pull request merged**